### PR TITLE
Update export responses + allow edits articles

### PR DESCRIPTION
**Pull request by aviv**
for the changes to the select responses UI

closes https://github.com/dobtco/knowledge_base/issues/203

### CODE REVIEW

File: contents/articles/screendoor/responses/exporting_responses.md [hunk 1]
```diff
@@ -8,16 +8,24 @@ app_pages:
...
-From the "Export responses" popup, select which responses you'd like to export. You can also choose to export ratings and blind fields (if you have the appropriate permissions). Then, select the format you'd like to export as and click "Download" to begin the export process.
+From the "Export responses" popup, select a format and optionally include blind fields and ratings (if you have the appropriate permissions). Then, click "Download" to begin the export process.
 
 ![export responses...](../images/export_responses_modal.png)
 
 Depending on the format you've selected, Screendoor may email you a link to download your exported responses. If it does, just click the link in the email to download your responses.
 
+## Exporting all responses
+
+To export all responses, first select all responses by checking the topmost box and then clicking the "Select all results" link if it appears.
```
Comment (ajb) [inline contents/articles/screendoor/responses/exporting_responses.md]: "if it appears" makes it sound like we don't know what our application is going to do...

**Comment by ajb** [comment]
:+1: just the one comment above.

**Comment by aviv** [comment]
:white_check_mark:

**Comment by ajb** [comment]
Cool, lgtm

**Pull request merged**